09_smoke_basin/9_p1_p2.py

```diff
@@ -11,24 +11,35 @@ class HeightMapAnalyzer():
     
     Additional helper functions calculate risk level and basin size to find the solution for parts 1 and 2.
 
-    Class attributes:
+    Class member variables:
     input_file (str)
-    heightmap (list of lists)
+    heightmap (list of list of ints)
+    low_points (list of tuples (row, column))
     """
 
-    def __init__(self, heightmap=None):
-        self.heightmap = heightmap
+    def __init__(self, file_name):
+        self.heightmap = self.CalcHeightmap(file_name)
+        self.low_points = self.CalcLowPoints()
 
-    """
-    Public functions:
-    """
-        
+    def CalcHeightmap(self, file_name):
```

Comment:
There's an error inside this function (not due to variable naming).  Try running on a small input.